packages/actions/input-edit/src/modules/inputEditTable/actions.js

```diff
@@ -73,3 +74,11 @@ export const setDataLoadingInProgress = dataLoadingInProgress => ({
     dataLoadingInProgress
   }
 })
+
+export const setCalculating = (inputData, isAdded) => ({
```

Comment:
question (non-blocking): Why have you called the second parameter `isAdded` instead of `loading` or similar?